src/frontend/src/hooks/useUser.ts

```diff
@@ -0,0 +1,23 @@
+import axios from "axios";
+import { useQuery } from "react-query";
+
+
+interface UserMeType {
+    login: string
+    avatar_url: string
+    html_url: string
+    type: "User"
+    blog: string
```

Comment:
Ich kann mir unter den Namen "login", "html_url", "blog" nichts wirklich vorstellen. Was bekommt man da?

Reply:
das naming kommt in einem späteren PR. (in der endpoint `/raw/me` muss dann angepasst werden (auf den provider) (dann benenne ich die auch noch um)